### DISCUSSION TITLE
normalize in sionna.channel.cir_to_ofdm_channel

### DISCUSSION THREAD

**Discussion by amira123bd** [NVlabs/sionna#420] [Q&A]
When obtaining the OFDM channel response, there's a parameter called "normalize" which can be toggled between True and False.  [sionna.channel.cir_to_ofdm_channel(frequencies, a, tau, normalize=False)], setting normalize to True or False affects the calculation. However, I'm unsure of the exact intention or effect of this parameter.

**Comment by SebastianCa** [comment]
Hi @amira123bd,

In this context, normalization means that we normalize each batch example and link so that the energy per resource grid is one. This involves averaging over TX antennas, RX antennas, OFDM symbols, and subcarriers.

You can think of it as a form of perfect power control from link adaptation (usually implemented 
by higher layer protocols which is not considered in Sionna).

I hope this answers your question.
